spec/models/user/email_spec.rb

```diff
@@ -42,5 +42,16 @@
         expect(subject.errors[:email].count).to eq(1)
       end
     end
+
+    context 'when the email is not confirmed' do
+      subject { create(:user_email, :unconfirmed) }
+
+      it 'sends email with ActiveJob queue' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery)
```

Comment:
Tried
```context 'when the email is not confirmed' do
      subject { create(:user_email, :unconfirmed) }

      it 'sends email with ActiveJob queue' do
        ActiveJob::Base.queue_adapter = :test
        expect{subject}.to have_enqueued_job.on_queue("mailers")
      end
    end```

but it didn't enqueue anything. I think we came up with this check to see if `deliver_later` is called because the jobs queue method didn't work.